tests/libtest/lib1301.c

```diff
@@ -21,16 +21,7 @@
  * SPDX-License-Identifier: curl
  *
  ***************************************************************************/
-#include "test.h"
-
-#define fail_unless(expr, msg)                             \
-  do {                                                     \
-    if(!(expr)) {                                          \
-      fprintf(stderr, "%s:%d Assertion '%s' failed: %s\n", \
-              __FILE__, __LINE__, #expr, msg);             \
-      return TEST_ERR_FAILURE;                             \
-    }                                                      \
-  } while(0)
+#include "curlcheck.h"
```

Comment:
Would it be "cleaner" to move `curlcheck.h` to `tests/libtest`?
The variable `unitfail` that it's using, is declared there already.

It may avoid the two way dependency between units and libtests.

Reply:
> Would it be "cleaner" to move curlcheck.h to tests/libtest?

It is used by just about every unit test and this the first change making use of it in libtests. It seemed to belong there more than in libtests...

Reply:
That's true, but it doesn't work without the global var in `libtests/first.c` and `libtests/test.h`.
It's somewhat confusing. Maybe that would be better be moved inside units and use local macro here.